.travis.yml

```diff
@@ -18,16 +19,15 @@ branches:
 
 before_install:
  - wget https://github.com/satooshi/php-coveralls/releases/download/v0.7.0/coveralls.phar
- - wget -O phpunit.phar https://phar.phpunit.de/phpunit-4.8.27.phar
 
 install:
-  - composer self-update
-  - composer install
+  - travis_retry composer self-update
+  - travis_retry composer install
```

Comment:
It's a common practice to avoid timeouts: https://docs.travis-ci.com/user/common-build-problems/